lib/awful/spawn.lua

```diff
@@ -1,4 +1,5 @@
 ---------------------------------------------------------------------------
+-- luacheck: no check comment
```

Comment:
That one will end up in the doc, right? If it does, that's not good... I guess the whole file will need to have a luacheck suppression comment...